tests/unit/pubsub_v1/publisher/test_publisher_client.py

```diff
@@ -28,6 +29,11 @@
 from google.cloud.pubsub_v1.publisher import exceptions
 from google.cloud.pubsub_v1.publisher._sequencer import ordered_sequencer
 
+import opentelemetry
+
+# Disable OpenTelemetry for publisher tests
+sys.modules["opentelemetry"] = None
```

Comment:
Caution - this modifies the global state, i.e. the `sys.modules` dictionary, and the change is not restored after the publisher tests are run. Let's make sure that the fixture is undone when not needed anymore.